Formula/hurl.rb

```diff
@@ -23,11 +24,12 @@ class Hurl < Formula
 
   on_linux do
     depends_on "pkg-config" => :build
-    depends_on "openssl@1.1"
+    depends_on "openssl@3"
   end
 
   def install
-    ENV["OPENSSL_DIR"] = Formula["openssl@1.1"].opt_prefix if OS.linux?
+    ENV["OPENSSL_DIR"] = Formula["openssl@3"].opt_prefix
+    ENV["OPENSSL_NO_VENDOR"] = "1"
```

Comment:
this should be only for `on_linux`?

Reply:
No. Let's set it unconditionally in case macOS builds start using the OpenSSL crate too. This will probably make them error out.

Reply:
👍